packages/kit/test/apps/basics/src/routes/anchor-with-manual-scroll/anchor.svelte

```diff
@@ -11,6 +11,7 @@
 	<p id="go-to-element">The browser scrolls to me</p>
 </div>
 <p id="abcde">I take precedence</p>
+<div />
```

Comment:
Note: I added this here our extra checks now brings a false positive to this case, where if the `onMount()` scrolls to the bottom of the page, we assumed that no scrolling happened.

Unless we can find a different check, this would be a known limitation for now. FWIW, if the `onMount()` scrolls to the top of the page, we had also assumed no scrolling happened in the past.

Extra note: "no scrolling happened" also means "run our custom scroll handling"